examples/download-progress/README.md

```diff
@@ -0,0 +1,8 @@
+# Download Progress
```

Comment:
I'd add a very short description explaining what the reader can learn from the example, i.e. "This example demonstrates how to integrate Bubble Tea with an `io.TeeReader` via `Program#Send()` (with links on the terms to their GoDocs).

Reply:
Added